### PR TITLE
[FRONT] #572 display create form after room editing

### PR DESCRIPTION
**Pull request by ntchou**
- fix startAt in function when editing
- add link to copy invitation
- add snackbar on copy link conversation and invitation
- fix room creation (crash)

### CODE REVIEW

**Review by HugoGresse — APPROVED** [review]

**Comment by ntchou** [comment]
@MattiaPrimavera ?

File: src/pages/AdminDashboard/AdminDashboard.tsx [hunk 1]
```diff
@@ -58,8 +56,7 @@ const AdminDashboard = () => {
     const onFormSubmit = (room, isEditing, remindAt) => {
         room.startAt = room.startAt / 1000
         if (isEditing) {
-            console.log('SAving room: ', room)
-            dispatch(editRoom(t, room))
+            dispatch(editRoom(t, room)).then(() => onCreateFormReset())
```
Comment (MattiaPrimavera) [inline src/pages/AdminDashboard/AdminDashboard.tsx:59]: async/await

File: src/pages/AdminDashboard/CreateRoomForm/formValidation.tsx [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+                ? new Date(originalValue)
+                : originalValue
+
+            return formatDate(date)
         })
-        .nullable(),
+        //min yup function doesn't work as expected. Use test instead
+        .test('startAt', i18n.t('invalid-min-date'), (value: Date) => {
+            return !value ? true : value >= formatDate(new Date())
+        }),
```
Comment (MattiaPrimavera) [inline src/pages/AdminDashboard/CreateRoomForm/formValidation.tsx:23]: is clicking on `Now` (`startAt` = null) taken into account by this validation logic ?

**Review by MattiaPrimavera — CHANGES_REQUESTED** [review]

File: src/pages/AdminDashboard/CreateRoomForm/formValidation.tsx (hunk 2 above)
Comment (ntchou) [reply src/pages/AdminDashboard/CreateRoomForm/formValidation.tsx:23]: Yes, you have a transform process before which return null if no value is provided

File: src/pages/AdminDashboard/AdminDashboard.tsx (hunk 1 above)
Comment (ntchou) [reply src/pages/AdminDashboard/AdminDashboard.tsx:59]: you're right. I'll fix it

**Review by HugoGresse — APPROVED** [review]

**Comment by ntchou** [comment]
@HugoGresse can you merge?

**Pull request merged**